extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/exporter/otlp/OtlpRecorder.java

```diff
@@ -106,7 +106,7 @@ public void accept(CompressionType compression) {
                     }
                 });
 
-        if (!exporterRuntimeConfig.traces().protocol().orElse("").equals(HTTP_PROTOBUF)) {
+        if (!exporterRuntimeConfig.traces().protocol().orElse("").equals(GRPC)) {
```

Comment:
The message is legacy, we currently allow either GRPC or HTTP_PROTOBUF. The exception should mention that.
We need to make sure we have a test for each protocol value.

Reply:
Isn't HTTP_PROTOBUF only available from Quarkus 3.3? I only saw `OtlpGrpcSpanExporter`, but no HttpExporter in 3.2

Reply:
HTTP_PROTOBUF and the HttpExporter are only available after 3.3

Reply:
But the PR is for 3.2. Should the message of the exception then contain something like `Only the GRPC Exporter is currently supported. HTTP_PROTOBUF is available after Quarkus 3.3.`?

Reply:
I didn't realise it was against the branch. Please re-do the PR against `main` and we will mark it for backporting until 3.2... This will affect many branches because main is already preparing for 3.5.
I propose this more generic message: `Only the "grpc" and "http/protobuf" values are supported in the otel.exporter.otlp.traces.protocol property. "http/protobuf" is only available after Quarkus 3.3`.

Reply:
The problem described in the issue doesn't exist anymore in `main`. It only exists in Quarkus 3.2 and lower versions. In 3.3 there was already [a rewrite of this](https://github.com/quarkusio/quarkus/blob/3.3/extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/exporter/otlp/OtlpRecorder.java#L88).